server/helpers/validation.js

```diff
@@ -0,0 +1,20 @@
+import Joi from '@hapi/joi';
+
+const validateUserSignup = data => {
+  const schema = Joi.object({
+    first_name: Joi.string().required(),
+    last_name: Joi.string().required(),
+    email: Joi.string()
+      .email()
+      .required(),
+    phone_number: Joi.string().required(),
```

Comment:
Would mind adding enforcing this validation by adding a regex to avoid wrong phone number input?? since for now, I can send letters and it will work